src/metakb/cli.py

```diff
@@ -215,11 +225,24 @@ def _help_msg(msg: str = "") -> None:
 
 def _load_normalizers_db() -> None:
     """Load normalizer DynamoDB database source data."""
+    aws_env_map = {
+        "Disease": (DISEASE_SKIP_AWS_DB_ENV_NAME, DISEASE_AWS_ENV_VAR_NAME),
+        "Therapy": (THERAPY_SKIP_AWS_DB_ENV_NAME, THERAPY_AWS_ENV_VAR_NAME),
+        "Gene": (GENE_SKIP_AWS_DB_ENV_NAME, GENE_AWS_ENV_VAR_NAME),
+    }
+
     for name, update_normalizer_db_fn in [
         ("Disease", update_normalizer_disease_db),
         ("Therapy", update_normalizer_therapy_db),
         ("Gene", update_normalizer_gene_db),
     ]:
+        skip_aws_db_env_name, aws_env_var_name = aws_env_map[name]
+        if environ.get(skip_aws_db_env_name) == "true" and aws_env_var_name in environ:
```

Comment:
does `skip_aws_db_env_name` matter? From the message below, it seems like we want to prohibit updating the cloud DBs from the metakb CLI regardless.

Reply:
Ya it probably is easiest to remove it than manually select y/n